dotnet/Data/CybersourceConstants.cs

```diff
@@ -127,6 +127,9 @@ public static class ManifestCustomField
             public const string Accept = "Accept";
             public const string Decline = "Decline";
             public const string Pending = "Pending Review";
+            public const string CaptureDelay = "Capture Delay";
+            public const string CaptureDelayInterval = "Capture Delay Interval";
+            public const string DecisionMangerInUse = "Using Decision Manger";
```

Comment:
```suggestion
            public const string DecisionManagerInUse = "Using Decision Manager";
```
Note the Manger/Manager typo is present in other files as well